app/User.php

```diff
@@ -240,6 +253,25 @@ public function getIsActiveAttribute()
         return self::where('id', $this->id)->active()->count() != 0;
     }
 
+    /**
+     * Get the access_active flag for the User.
+     *
+     * @return bool
+     */
+    public function getIsAccessActiveAttribute()
+    {
+        if (self::where('id', $this->id)->accessActive()->count() != 0) {
+            return true;
+        }
+        foreach ($this->teams()->get() as $team) {
```

Comment:
Is there another way that isn't a foreach loop to do this?

Reply:
You could probably do it in SQL but it would be gross. I can try it if you want.

Reply:
Instead of fetching the entire model of each team, can you just get an array of team `id`s and do an `array_intersect()` with the access teams from config?

Reply:
Resolving this thread as we're no longer using teams for access control